lib/presentation/widget/home/home_viewmodel.dart

```diff
@@ -117,17 +118,27 @@ class HomeViewModel extends BaseViewModel {
 
   void cancelSearchState() {
     store.dispatch(DisableSearchStateAction());
-    if (store.state.uiState.searchState.searchDestination == SearchDestination.mySpace) {
+    if (store.state.uiState.searchState.searchDestination == SearchDestination.MY_SPACE) {
       store.dispatch(MySpaceAction(Right(DisableSearchViewState())));
       store.dispatch(CleanMySpaceStateAction());
     }
+
+    if (store.state.uiState.searchState.searchDestination == SearchDestination.SHARED_SPACE) {
```

Comment:
pls use `else if (store.state.uiState.searchState.searchDestination == SearchDestination.SHARED_SPACE)` to optimize code